--- openssl/crypto/bio/bss_log.c	2023-04-10 11:30:52.432730124 -0400
+++ openssl-3.1.0/crypto/bio/bss_log.c	2023-04-10 03:25:51.171181966 -0400
@@ -42,7 +42,16 @@
 #elif defined(__DJGPP__) && defined(OPENSSL_NO_SOCK)
 # define NO_SYSLOG
 #elif (!defined(MSDOS) || defined(WATT32)) && !defined(OPENSSL_SYS_VXWORKS) && !defined(NO_SYSLOG)
-# include <syslog.h>
+//# include <syslog.h>
+#  define LOG_EMERG       0
+#  define LOG_ALERT       1
+#  define LOG_CRIT        2
+#  define LOG_ERR         3
+#  define LOG_WARNING     4
+#  define LOG_NOTICE      5
+#  define LOG_INFO        6
+#  define LOG_DEBUG       7
+#  define LOG_DAEMON      (3<<3)
 #endif
 
 #include <openssl/buffer.h>
@@ -198,7 +207,7 @@
     if (inl < 0)
         return 0;
     if ((buf = OPENSSL_malloc(inl + 1)) == NULL) {
-        ERR_raise(ERR_LIB_BIO, ERR_R_MALLOC_FAILURE);
+        // ERR_raise(ERR_LIB_BIO, ERR_R_MALLOC_FAILURE);
         return 0;
     }
     memcpy(buf, in, inl);
@@ -391,9 +400,9 @@
 static void xopenlog(BIO *bp, char *name, int level)
 {
 #  ifdef WATT32                 /* djgpp/DOS */
-    openlog(name, LOG_PID | LOG_CONS | LOG_NDELAY, level);
+//    openlog(name, LOG_PID | LOG_CONS | LOG_NDELAY, level);
 #  else
-    openlog(name, LOG_PID | LOG_CONS, level);
+//    openlog(name, LOG_PID | LOG_CONS, level);
 #  endif
 }
 
